examples/performance/01_gpu.ipynb

```diff
@@ -172,7 +172,7 @@
VoBrUbfqdQ6o0PbaEWNxN/P3kZjbTIXX0Xnef1Py5lTP+0xP4kEQP309LBXee2s3UJ/l5u1ZDTznx2eo9R+wGSxJdIri/yaEDQ+qwGN/PxBkm2sBOEb5r17vIQNeswVngD+SAAAITElEQVT4LrbYYosttti5dqx6BhyAiPFD4xP+CAi/AZXzm2cj3nCMexCQLPaObK5SBhyvltljckepBpmL1xT+mrZfi7tpVRiw4H+S/ZmKvd1kM/7OM4G6PnBw/yv1vs88PnuMxd693UDpl5tH6M+YAcBArgLL48ZqUN/N1H7OQm+j+e3AYVV49Lo4vP+2n9fsGO1/Ck5vovzXr1VXekePz6+Vh5QYWoDvYosttthii93WpifxY0HX1Gbo0W/UzgXR9yQIWeyO7A3T+GdBsJn1Bttz54L9XF07w+bE3YBDn54875XB7sxzjh5zsbuzWyR0AByM9QJwWqUfyL3AI2p/xGRcXaH3jwTfzNyhL91E8S8fccbfXoHuP3qtm6q7c6+Dh7tOFuC72GKLLbbYYq9rNwHh/LzXr+6+kt2jwGOxe2SvQeMfAYfcCzmh66uS7uj/jq2N+nlTEHwM7FZ2IyjIT3wNoCsHuvG9LHZHdo4/nwLBUz+u/PfozHb7vzl6P3Ca4m8jjc7+gGqnmAhz4BaYZ0HccgLAQwW7tS3Ad7HFFltsscXetJ0LhN/V8Rdb7CY7h8Zf96Ofqgbbc+coyecUdm+rcA7Mg1xgNjA/yrhYAO/7Y6cqwMCsL4/8eOK/s9Vg4DS9HzhN8a+rxefaEd89FH47Tfc/+J8TYPp9Sg4twHexxRZbbLHF3rbd40BgscVGdqxqBhwNdjnNodkZYbdy91u11+55z89b1u2DtlMJyBtZDRP/nZvZDgAxHYBXqfqOHZ3PofKfq+o8Z7cdb5efdAYDYuZ5kwPd9O7ujS3Ad7HFFltsscUWW2yxQ5sLaG8Sdjt46JYzet+U3bat4AEF74u9ot02qfMqSv1TYJyfYz3Db5j984pJnldW/s/PeZjrZQG+iy222GKLLbbYYoudZ8cC3vvW137MHmjAvtgbtnOSOidECo8DxyNMh/HDb93OElB8hMwHmhUDeFsHI/o6gGsA33hnBz20b73D49/lsR/78f9GZv7r7ujYi+8vx7/rz35n/r/4/nL8Oz7+Y9/7H/Nv/9iP/9h9H3jcv/9dH/+uP/us/79T4AsARPTTzPxb3ulB78nxH/Nnvw/Hv2u768+/HP/xrv27trv+/MvxH/fx79oe896zHH/x/cf8/T/m49/1Zz9md9R4sdhiiy222GKLLbbYYosttthi78YW4LvYYosttthiiy222GKLLbbYe213AXx/9A6OeV+O/5g/+304/l3bXX/+5fiP89j3we768y/Hf9zHv2t7zHvPcvzHbXf9+ZfjP85jH7V33uO72GKLLbbYYosttthiiy222GLv0haq82KLLbbYYosttthiiy222GLvtb0z4EtEv5OI/m8i+kUi+uF3cLxvJ6IvENH/RUS/QER/QO//FiL600T0V/Tys2/5fXgi+otE9D/q7e8gop/S7+G/JaLVWzz2h0T0J4noLxPRXyKi3/ouPz8R/ZB+9z9PRP8NEW3e5ee/L7b4/uL7j9X3gcfp/4vvL74PPE7f1+M9Sv9ffL/Y4vuPy/f1+A/C/98J8CUiD+A/BvD9AL4TwD9FRN/5lg8bAPxBZv5OAN8D4F/SY/4wgD/DzH8LgD+jt9+m/QEAf6m6/e/g/2/vbkKsKuMAjD9/mhQ0KG0hkxOYJLYI1Gih5EL6IJHITYughQv30ioYWrUPP1ZtihYRBJmkuDDoY20pRIgfaRg54teiElwV/Vucd5g7MlCI5/Xe8z4/ODDn3OG+933nuRfOzLl34EBmPg38DuztcexDwInMfAbYVB5HlflHxFpgH/B8Zj4LPAS8Sd35P3C2b/s02j403b/t236r7UOD/dv+Attvq32YsP4zs/cN2AZ8NbI/C8zWGHtkzKPAK8AFYLocmwYu9DjmDF1oLwLHgaD7Z85TS63LfR77UeAy5X3cI8erzB9YC1wBVgNTZf6v1pr/uGy2b/uttr/UPFvo3/Ztf6l5ttB+uf8m+7f9RWth+w21X+57Yvqvdanz/ILMmyvHqoiIdcAW4CSwJjOvlZuuA2t6HPog8A7wT9l/HPgjM/8u+32uw1PALeDjctnFhxGxkkrzz8yrwPvAb8A14E/gNPXmPy5sv2P77bUPbfZv+7YPbbYPjfZv+4vYfqeJ9mGy+h/8h1tFxCPAF8DbmXl79LbsfgXRy8daR8RrwM3MPN3H/f8PU8BzwAeZuQW4w12XOPQ8/1XAbron4xPASmBnH2NpabZv+y17EP3bvu2PA1/76/dv++PB9n3t/y+1TnyvAk+O7M+UY72KiIfpngCfZuaRcvhGREyX26eBmz0N/wLwekT8CnxGd+nDIeCxiJgq39PnOswBc5l5suwfpntS1Jr/y8DlzLyVmX8BR+jWpNb8x4Xt236r7UN7/du+7c9rrX1ou3/bX2D7bbUPE9R/rRPfH4AN5dO9ltG94flYnwNGRAAfAecyc//ITceAPeXrPXTvA7jvMnM2M2cycx3dfL/NzLeA74A3Kox/HbgSERvLoZeAs1SaP93lDlsjYkX5WcyPX2X+Y8T2bb/V9qGx/m3f9kc01T4037/tL7D9ttqHSer/Xt4YfC8bsAv4GfgFeLfCeNvp/qT/E/Bj2XbRXXP/DXAR+BpYXeGx7ACOl6/XA98Dl4DPgeU9jrsZOFXW4EtgVc35A+8B54EzwCfA8przH5fN9m2/1fbLWjTZv+3bfqvtl8fSXP+2v2gtbD/bab+MPxH9R3mwkiRJkiQN0uA/3EqSJEmS1DZPfCVJkiRJg+aJryRJkiRp0DzxlSRJkiQNmie+kiRJkqRB88RXkiRJkjRonvhKkiRJkgbNE19JkiRJ0qD9C+HJjXGzJ4sIAAAAAElFTkSuQmCC\n",
```

Comment:
Can we clear output before committing Jupyter notebooks as some of these are huge. It will make a happier sized repo as well as less insane diff's.